tools/print-change-list.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+
+import argparse
+import re
+from subprocess import run
+import textwrap
+from os import getenv
+from subprocess import PIPE
+from sys import exit, stderr
+
+import dateutil.parser
+import requests
+
+GITHUB_API_URL = 'https://api.github.com'
+
+
+class PRInfo(object):
+    def __init__(self, id, title):
+        self.id = id
+        self.title = title
+
+
+def pr_link(repo, id):
+    """Return a link to a GitHub Pull Request in markdown format"""
+
+    return '[#{id}](https://github.com/{repo}/pull/{id})'.format(id=id, repo=repo)
+
+
+def get_last_tag():
+    proc = run(['git','tag','--list','--sort=-taggerdate'],
+        stdout=PIPE)
+    tags = proc.stdout.decode().splitlines()
```

Comment:
If and when a user with a non-UTF-8 terminal hits that issue, they can fix it :P